centreon/www/class/centreonDowntime.class.php

```diff
@@ -415,52 +412,39 @@ public function getForEnabledServices()
                 h.host_name,
                 s.service_id,
                 s.service_description
-            FROM downtime_period dtp,
-                downtime dt,
-                downtime_service_relation dtr,
-                service s,
-                host h,
-                host_service_relation hsr
-            WHERE dtp.dt_id = dtr.dt_id
-                AND dtp.dt_id = dt.dt_id
-                AND dtr.service_service_id = s.service_id
-                AND hsr.service_service_id = s.service_id
-                AND hsr.host_host_id = h.host_id
-                AND h.host_id = dtr.host_host_id
-                AND s.service_activate = '1'
-            UNION
-                SELECT dt.dt_id,
-                    dt.dt_activate,
-                    dtp.dtp_start_time,
-                    dtp.dtp_end_time,
-                    dtp.dtp_day_of_week,
-                    dtp.dtp_month_cycle,
-                    dtp.dtp_day_of_month,
-                    dtp.dtp_fixed,
-                    dtp.dtp_duration,
-                    s.service_description as obj_name,
-                    dtr.service_service_id as obj_id,
-                    h.host_name as host_name,
-                    h.host_id
-                FROM downtime_period dtp,
-                    downtime dt,
-                    downtime_service_relation dtr,
-                    service s,
-                    host h,
-                    hostgroup_relation hgr,
-                    host_service_relation hsr
-                WHERE
-                    dtp.dt_id = dtr.dt_id
-                    AND dtp.dt_id = dt.dt_id
-                    AND dtr.host_host_id = h.host_id
-                    AND hsr.hostgroup_hg_id = hgr.hostgroup_hg_id
-                    AND hgr.host_host_id = h.host_id
-                    AND s.service_id = hsr.service_service_id
-                    AND dtr.service_service_id = s.service_id
-                    AND h.host_activate = '1';
+            FROM downtime_period dtp
+            INNER JOIN downtime dt ON dtp.dt_id = dt.dt_id
+            INNER JOIN downtime_service_relation dtr ON dtp.dt_id = dtr.dt_id
+            INNER JOIN service s ON dtr.service_service_id = s.service_id
+            INNER JOIN host_service_relation hsr ON hsr.service_service_id = s.service_id
+            INNER JOIN host h ON hsr.host_host_id = h.host_id AND h.host_id = dtr.host_host_id
```

Comment:
To be more readable.
```suggestion
            INNER JOIN downtime dt
                ON dtp.dt_id = dt.dt_id
            INNER JOIN downtime_service_relation dtr
                ON dtp.dt_id = dtr.dt_id
            INNER JOIN service s
                ON dtr.service_service_id = s.service_id
            INNER JOIN host_service_relation hsr
                ON hsr.service_service_id = s.service_id
            INNER JOIN host h
                ON hsr.host_host_id = h.host_id
                AND h.host_id = dtr.host_host_id
```